html_css/v2/main-course/positioning.md

```diff
@@ -0,0 +1,38 @@
+### Positioning
+
+You have learned various ways how to position elements on the screen and even though the methods you know so far are sufficient in most cases, these CSS rules are still very powerful and will certainly help you step up your CSS game!
+
+### Learning Outcomes
+ - You'll learn how to use absolute positioning.
+ - You'll learn how to use fixed positioning.
+ - You'll learn how to use sticky positioning.
+ - You'll know the difference between each property and how to combine them.
+
+### Static and Relative Positioning
+The difference between static and relative is fairly simple. Static is the default position of every element, and properties _top_, _right_, _bottom_, and _left_ have no affect on statically positioned elements. Relative on the other hand is pretty much the same as static, but properties  _top_, _right_, _bottom_, and _left_ affect the specific element relative to itself.
```

Comment:
If you change the intro above, I'd edit the wording here to reflect the fact that we've already mentioned that there is a default positioning method... so you can start with something like:

> The default positioning mode that you've gotten used to is `position: static`.....

and then continue from there.

also: I don't think  you need to mention `top, right, bottom, left` twice here.... you can simplify by just saying something like:

> Relative, on the other hand, is pretty much the same as static except that you can move the element around using the properties top, ri....(etc.)

unrelated note: feel free to put all of these suggestions into your own words.  Or quote them.  I don't care either way 💜